nav2_dwb_controller/dwb_plugins/include/dwb_plugins/kinematic_parameters.hpp

```diff
@@ -74,6 +75,30 @@ struct KinematicParameters
 
   inline double getMinSpeedXY_SQ() {return min_speed_xy_sq_;}
   inline double getMaxSpeedXY_SQ() {return max_speed_xy_sq_;}
+  
+  /**
+   * @brief Check to see whether the combined x/y/theta velocities are valid
+   * @return True if the magnitude hypot(x,y) and theta are within the robot's
+   * absolute limits
+   *
+   * This is based on three parameters: min_speed_xy, max_speed_xy and
+   * min_speed_theta. The speed is valid if 1) The combined magnitude hypot(x,y)
+   * is less than max_speed_xy (or max_speed_xy is negative) AND 2) min_speed_xy
+   * is negative or min_speed_theta is negative or hypot(x,y) is greater than
+   * min_speed_xy or fabs(theta) is greater than min_speed_theta.
+   */
+  inline bool isValidSpeed(double x, double y, double theta)
+  {
+    double vmag_sq = x * x + y * y;
+    if (max_speed_xy_ >= 0.0 && vmag_sq > max_speed_xy_sq_ + std::numeric_limits<float>::epsilon())
```

Comment:
Readd all brackets, this will not be accepted without them

Reply:
ament uncrustify did the trick, thanks!